### PR TITLE
Added explanatory text to get pet activity layout and changed icon to…

### PR DESCRIPTION
**Pull request by GretaGri**
… one house icon.

### CODE REVIEW

**Comment by vycius** [comment]
XML Vector drawable can be used instead of `rodykles_baltos.png`. 
See https://materialdesignicons.com/ `chevron-double-down`

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <string name="my_story">Mano istorija</string>
     <string name="no_more_pets_to_swipe">O ne!\nSąrašas jau baigėsi.\nPatikrink pamėgtų gyvūnų sąrašą</string>
     <string name="empty_favorites_list">Neturite pamėgtų gyvūnų,\nlaikas išssirinkti draugą!</string>
+    <string name="spausk_susisiek">Spausk ir susisiek\nsu prieglaudos atstovu!</string>
```
Comment (vycius) [inline app/src/main/res/values/strings.xml]: Let's use english variable names.

Comment (vycius) [reply app/src/main/res/values/strings.xml]: What would you think about adding more information about mentorship ? E.g `Spausk ir susisiek\nsu prieglaudos atstovu!\nNepamiršk paminėti apie GetPet ir gauk mentorių.`

**Review by vycius — APPROVED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (GretaGri) [reply app/src/main/res/values/strings.xml]: Good suggestions, followed both.

**Review by vycius — APPROVED** [review]

File: app/src/main/res/layout/activity_get_pet.xml [hunk 2]
```diff
@@ -67,13 +67,32 @@
...
+            android:textSize="@dimen/text_medium"
+            android:text="@string/push_and_contact"/>
+
+        <ImageView
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginBottom="16dp"
+            android:layout_gravity="center"
+            android:scaleType="centerInside"
+            android:src="@drawable/double_down" />
```
Comment (vycius) [inline app/src/main/res/layout/activity_get_pet.xml]: Let's use `app:srcCompat` with vector drawables. Fore more information why https://medium.com/androiddevelopers/appcompat-v23-2-age-of-the-vectors-91cbafa87c88

File: app/build.gradle [hunk 3]
```diff
@@ -19,6 +19,7 @@ android {
         versionCode major * 10000 + minor * 100 + patch
         versionName "${major}.${minor}.${patch}"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables.useSupportLibrary = true
```
Comment (vycius) [inline app/build.gradle]: Possibly no need for that as we are targeting quite new Android version and vector drawables are used all over the app.

**Review by vycius — APPROVED** [review]
Just remove `vectorDrawables.useSupportLibrary` and let's merge.

**Pull request merged**